deployment/playbooks/provisioning.yml

```diff
@@ -3,15 +3,17 @@
   # become: yes
   user: root
   roles:
+    # server setup
     - { role: network }
     - { role: storage }
     - { role: base, tags: ['base'] }
+    - { role: motd, tags: ['motd'] }
+
+    # services
     - { role: mattermost, tags: ['mattermost'] }
     - { role: matterbridge, tags: ['matterbridge'] }
-    - { role: motd, tags: ['motd'] }
-    - { role: docker, tags: ['docker'] }
-    - { role: watchtower, tags: ['watchtower'] }
```

Comment:
Why remove docker and watchtower? This way we can't easily change watchtower configuration without having to redeploy any service that depends on watchtower.

Reply:
You're right. Didn't thought about that.